### ISSUE TITLE
Name of the Repository is misleading

### ISSUE THREAD

**Issue by DDhanushka** [AiSERTIVE-INCEPTION/Everything-Is-Connected#1]
As I know we are going to make a repository for opensource JS, CSS, HTML design codes. **This name doesn't give a clue to any user.** Please change the name to something that can give the goal of our idea. (I mean who will search for DARK in github) @anjana-dodampe

**Comment by anjana-dodampe** [comment]
I agree with you. But what I mean from repo Everything-is-connected is simple final outputs with combination of these 3 elements
